common/swagger/swagger.yaml

```diff
@@ -17,126 +17,96 @@ schemes:
   - https
   - http
 paths:
-  /models:
+  /model/id:
     get:
       tags:
-        - models
-      summary: Returns a list of models loaded in the server
-      description: Returns a list of models loaded in the server
-      operationId: getModels
+        - metadata
+      summary: Returns model id
+      description: Returns unique id of the model loaded in the server and used for scoring
+      operationId: getModelId
       produces:
         - application/json
```

Comment:
@osery Should this be text/plain since we're returning just the model-id string itself?

Reply:
Good point, thanks :pray:. Fixed.